pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -343,12 +343,12 @@ func (rc *reconciler) attachDesiredVolumes() {
 		klog.V(5).InfoS("Starting attacherDetacher.AttachVolume", "volume", volumeToAttach)
 		err := rc.attacherDetacher.AttachVolume(volumeToAttach.VolumeToAttach, rc.actualStateOfWorld)
 		if err == nil {
-			klog.InfoS("attacherDetacher.AttachVolume started", "volume", volumeToAttach)
+			klog.InfoS("attacherDetacher.AttachVolume started", "volume name", volumeToAttach.VolumeName, "node name", volumeToAttach.NodeName, "scheduled pods", klog.KObjSlice(volumeToAttach.ScheduledPods))
```

Comment:
Do we care about information for VolumeSpec?

Reply:
If the volume attachment fails then it would be helpful to let user know about the name of the volume, which node it failed to attach and which pod were going to use it.

Reply:
/lgtm